### ISSUE TITLE
Garbled error messages

### ISSUE THREAD

**Issue by grimbough** [Huber-group-EMBL/rhdf5#39]
When `rhdf5::h5errorHandling('verbose')` is set the file and function names are garbled.  Why is this?

```
Error in H5Fopen(file) : libhdf5
    error #000: ����)V in �"$�)V(): line 509
        class: HDF5
        major: File accessibilty
        minor: Unable to open file
    error #-04:  in (): line 1400
        class: HDF5
        major: File accessibilty
        minor: Unable to open file
    error #-03: @;$�)V in (): line 1546
        class: HDF5
        major: File accessibilty
        minor: Unable to open file
    error #-02: p�ξ)V in ���)V(): line 734
        class: HDF5
        major: Virtual File Layer
        minor: Unable to initialize object
    error #-01: ����)V in (): line 346
        class: HDF5
        major: File accessibilty
        minor: Unable to open file
```

**Comment by grimbough** [comment]
A nice explanation from HDF5 staff at https://forum.hdfgroup.org/t/custom-error-messages-and-behaviour-of-h5eget-msg/5784

**Comment by grimbough** [comment]
Should be fixed in **rhdf5** 2.29.2. 

```r
> H5Fopen('foo.h5')
Error in H5Fopen("foo.h5") : libhdf5
    error #000: H5F.c in H5Fopen(): line 509
        class: HDF5
        major: File accessibilty
        minor: Unable to open file
    error #001: H5Fint.c in H5F__open(): line 1400
        class: HDF5
        major: File accessibilty
        minor: Unable to open file
    error #002: H5Fint.c in H5F_open(): line 1546
        class: HDF5
        major: File accessibilty
        minor: Unable to open file
    error #003: H5FD.c in H5FD_open(): line 734
        class: HDF5
        major: Virtual File Layer
        minor: Unable to initialize object
    error #004: H5FDsec2.c in H5FD_sec2_open(): line 346
        class: HDF5
        major: File accessibilty
        minor: Unable to open file
```
